components/FlexibleForms/StepForm.tsx

```diff
@@ -72,8 +72,11 @@ const StepFormInner = ({
   const { saved, setSaved } = useAutosave();
   const router = useRouter();
 
-  if (goBackToTaskList && saved && isValid)
+  if (goBackToTaskList && saved && isValid) {
     router.push(`/submissions/${router.query.id}`);
+  } else {
```

Comment:
For my own understanding, why use this syntax rather than a normal if else?

Reply:
very good point!